ruff.toml

```diff
@@ -1,13 +1,28 @@
+line-length = 90
+indent-width = 4
+
+[format]
+# Use same settings as Black.
+quote-style = "double"
+indent-style = "space"
+docstring-code-format = true
+skip-magic-trailing-comma = false
+line-ending = "auto"
+
 [lint]
-select = ["ALL"]
-ignore = [
-    "INP001",   # implicit-namespace-package
-    "COM812",   # TrailingCommaMissing
-    "COM819",   # TrailingCommaProhibited
+select = [
+    # pycodestyle
+    "E",
+    # pyupgrade
+    "UP",
```

Comment:
[What Python version are you targeting?](https://docs.astral.sh/ruff/settings/#target-version)